app/util/statistics.ts

```diff
@@ -41,31 +54,53 @@ export function groupBySymptoms(
     .filter(reportFilter)
     .map(report => report.symptoms);
 
-  const symptomStats = {
```

Comment:
Her foreslår jeg kanskje at vi bare setter opp hele listen først, så slipper vi så mye logikk for å sjekke om stat'en er opprettet enda eller ikke :)